base/student.rb

```diff
@@ -1,10 +1,10 @@
 require_relative 'person'
 
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :name, :classroom
 
   def initialize(age, name = 'Unknown', parent_permission: true, classroom: '')
-    super(name, age, parent_permission)
+    super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] in this class there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom).  please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Reply:
well resolved! ✔️